{"uuid":"482479e8-6b35-4a5a-8aa4-9e5e1aa65b3e","name":"Scenario Outline: The user getting error message with invalid python code for Traversal page","children":["7a2651dd-c3cc-4042-b7fa-4f38e762b275"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"ea65fe8b-1969-4a1d-9b95-41f1da5549c4-attachment","type":"image/png"},{"name":"Myscreenshot","source":"13b711a0-b3a3-45ab-9df8-28361a9087e2-attachment"}],"parameters":[],"start":1696883658654,"stop":1696883658695},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883658695,"stop":1696883658695},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883658696,"stop":1696883658696},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883658697,"stop":1696883658697}],"start":1696883658651,"stop":1696883658697}